src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -262,6 +262,10 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
             if (isset($this->aliases[$id])) {
                 $id = $this->aliases[$id];
             }
+            if (isset($this->privates[$id]) && !$this instanceof ContainerBuilder) {
```

Comment:
not sure.. https://github.com/symfony/symfony/blob/v3.2.8/src/Symfony/Component/DependencyInjection/ContainerBuilder.php#L577

but this definitely fixes tests :)